packs/classfeatures/precise-strike.json

```diff
@@ -25,6 +25,15 @@
             "title": "Pathfinder Advanced Player's Guide"
         },
         "rules": [
+            {
+                "key": "ActiveEffectLike",
+                "mode": "override",
+                "path": "flags.pf2e.swashbuckler.preciseStrike.value",
```

Comment:
Why `preciseStrike.value`?

Reply:
If I remember, at the time I thought it'd be a good idea to leave it open to PC2 adding stuff to Precise Strike, like dice scaling or whatnot. So no good reason, really. I'll fix it later today.